### PR TITLE
Messages for Vets.gov users about health tools moving to MHV

### PR DESCRIPTION
**Pull request by U-DON**
## Description
Added alerts to individual health tools about them moving to MHV.

## Testing done
Verified locally that the messages are rendering properly.

## Acceptance criteria
- [x] Health tools on Vets.gov should have messages about moving to MHV soon.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/rx/containers/Active.jsx [hunk 1]
```diff
@@ -270,7 +270,7 @@ class Active extends React.Component {
 
   render() {
     return (
-      <ScrollElement id="rx-active" name="active" className="va-tab-content">
```
Comment (dhall) [inline src/applications/rx/containers/Active.jsx]: Is this needed for your changes or unrelated cleanup?

File: src/applications/rx/containers/History.jsx [hunk 2]
```diff
@@ -207,7 +207,7 @@ class History extends React.Component {
 
   render() {
     return (
-      <ScrollElement id="rx-history" name="history" className="va-tab-content">
```
Comment (dhall) [inline src/applications/rx/containers/History.jsx]: Is this needed for your changes or unrelated cleanup?

**Review by dhall — APPROVED** [review]
LGTM

File: src/applications/rx/containers/Active.jsx (hunk 1 above)
Comment (U-DON) [reply src/applications/rx/containers/Active.jsx]: Yup, these changes were necessary for visual effect. Without moving the `va-tab-content` class one level up, the line under the Rx tabs would have been under the MHV moving notice instead of above and connected with the tabs.
